tests/api-testing/tests/test_foldersnew.py

```
@@ -0,0 +1,142 @@
+def retrieve_new_dashboard(session, base_url, org_id, folder_id):
+    resp_get_dashboard = session.get(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{folder_id}")
+    print(resp_get_dashboard.content)
+    assert (
+        resp_get_dashboard.status_code == 200
+    ), f"Retrieve this folder dashboard: Expected 200, but got {resp_get_dashboard.status_code} {resp_get_dashboard.content}"
+
+def delete_new_dashboard(session, base_url, org_id, folder_id):
+    resp_delete_dashboards = session.delete(
```

Ruff (0.8.2)

Local variable `resp_delete_dashboards` is assigned to but never used

Remove assignment to unused variable `resp_delete_dashboards`

(F841)